lib/src/maintenance.dart

```diff
@@ -552,3 +555,25 @@ Future<Maintenance> detectMaintenance(
 
 bool _isMarkdown(String fileName) =>
     p.extension(fileName).toLowerCase() == '.md';
+
+bool _hasExampleInReadmeMarkdown(String text) {
```

Comment:
Give this a doc comment...

Reply:
ie. explain what pulls out... give an example README... for how to put examples in READMEs...
```
/// Detects if the README [text] contains an example section.
///
/// ```
/// # Example
/// ....
/// ```
```